mtags/src/main/scala-2/scala/meta/internal/pc/Signatures.scala

```diff
@@ -138,52 +132,90 @@ trait Signatures { this: MetalsGlobal =>
       nameResolvesToSymbol(top.name.toTermName, top)
     }
 
-    def nameResolvesToSymbol(sym: Symbol): Boolean = {
-      nameResolvesToSymbol(sym.name, sym)
+    def isSymbolInScope(sym: Symbol, prefix: Type = NoPrefix): Boolean = {
```

Comment:
Changes here and the two methods below come from:
https://github.com/scalacenter/scalafix/blob/69c0369ab49c41b0473ca68fe0a8ade396d3157e/scalafix-rules/src/main/scala/scala/meta/internal/pc/ScalafixGlobal.scala#L382